cmd/gokey/main.go

```diff
@@ -54,7 +55,10 @@ func genSeed(w io.Writer) {
 }
 
 func genPass(seed []byte, w io.Writer) {
-	password, err := gokey.GetPass(pass, realm, seed, &gokey.PasswordSpec{10, 3, 3, 1, 1, ""})
+	if isInvalidLength(length) {
```

Comment:
let's do input validation in the `main` function, like the rest of the flags and call these helpers when we validated everything

Reply:
Resolved in 4dde8f5